Lab 04 tests/UnitTest1.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using Xunit;
+using Lab04_TicTacToe.Classes;
+using Lab04_TicTacToe;
+
+namespace Lab_04_tests
+{
+    public class UnitTest1
+    {
+        [Fact]
+        public void TestForAWinner()
+        {
+            Player player1 = new Player();
+            player1.Name = "jane";
+            player1.Marker = "O";
+
+            Player player2 = new Player();
+            player2.Name = "doe";
+            player2.Marker = "X";
+
+            Game newGame = new Game(player1, player2);
+
+            Board testBoard = new Board()
+            {
+                GameBoard =

```

Comment:
Getting errors from this. 
"Board.GameBoard cannot be assigned as an object initializer"